packages/cli-plugin-metro/src/commands/runPackager/index.ts

```diff
@@ -0,0 +1,18 @@
+import {runPackager} from './runPackager';
+
+export default {
+  name: 'run-packager-hook',
+  description: 'Starts the packager',
```

Comment:
For sure this should be named in better way, do you have any ideas how we could call this?

Reply:
`ensure-dev-server` / `ensure-packager`?

Reply:
Thanks @tido64 for proposition! We're mergining this logic to one command - `start`, so that everything will be in one place (see [here](https://github.com/react-native-community/cli/pull/2021#discussion_r1277710939)), and I will remove this command soon as it wouldn't make sense to keep it.